src/jrd/SysFunction.cpp

```diff
@@ -432,6 +432,12 @@ static const char
 	FALSE_VALUE[] = "FALSE",
 	TRUE_VALUE[] = "TRUE";
 
+// Get pages
+const char
+    PAGES_ALLOCATED[] = "PAGES_ALLOCATED",
+    PAGES_USED[] = "PAGES_USED",
+    PAGES_FREE[] = "PAGES_FREE";
+
 
```

Comment:
Why these constants put separately ? 
Its belongs to the "SYSTEM namespace: global and database wise items" section.